src/rust/engine/graph/src/entry.rs

```diff
@@ -334,6 +340,19 @@ impl<N: Node> Entry<N> {
           None,
           previous_result,
         ),
+        EntryState::Errored {
```

Comment:
Since this has the same shape as `NotStarted`, you should be able to merge the match arms as `NotStarted(..) | Errored(..) => ..`.

But, _also_ given that this has the same shape as `NotStarted`, maybe the better approach would be to rename `NotStarted` to some thing more neutral, and use it for both cases? I don't think that there is necessarily any value in the semantic label of "Errored".

Reply:
Changed to `Dormant`!